R/addAlpha.R

```diff
@@ -585,6 +585,19 @@ setMethod("addAlpha", signature = c(x = "SummarizedExperiment"),
         FUN("'faith' index can be calculated only for TreeSE with rowTree(x) ",
             "populated.", call. = FALSE)
     }
+    # Check for unsupported values (relative or negative)
+    if(any(assay(x) < 0)){
```

Comment:
assay(x) might not b the assay that user wants to use. --> use assay(x. assay.type)